data/json/items/melee/spear.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "id": "spear_stone",
```

Comment:
We need a knapping system before we add any stone weapons.

Reply:
Removed the crafting recipe, left the item in as a museum spawn.